### ISSUE TITLE
Hair physics not working

### ISSUE THREAD

**Issue by mgroening** [soupday/cc3_blender_tools#22]
Hi, I really like this add-on and I'm impressed with how well it works!
However, the hair physics just don't work. I tried everything. I choose to automatically generate physics when importing the character. Everything looks fine. But as soon as I start playing the animation the hair doesn't move (like there are no physics at all). Baking the physics in Blender has no effect. However, the hair physics works in iClone7 as intended. Baking the physics in iClone makes also no difference. Using different hairstyles also doesn't solve this issue. I use Blender 2.93.4 and iClone 7.92. This add-on is already up-to-date.
Did I perhaps miss something?

**Comment by mgroening** [comment]
If someone else has the same problem: My workaround is currently to render the physics simulation in CC and save it as alembic. Then I import it in Blender.

**Comment by stephankellermayr** [comment]
Due to the fact that IC/CC cannot export the mesh with the subdivision applied there, this workaround unfortunately makes no sense for me.
The error or the "missing feature" of a correctly working physics simulation therefore remains for me.

**Comment by raitan2008** [comment]
> If someone else has the same problem: My workaround is currently to render the physics simulation in CC and save it as alembic. Then I import it in Blender.

Yeah, that is what I do--just have to copy over the textures. I also sometimes import the same figure without physics, set up my cam, then import it again with physics without touching any cam settings, and rendering from there.
